typescript-conditionals/main.ts

```diff
@@ -0,0 +1,116 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+// 1.
+// function isUnderFive(number: number){
+//   if(number<5){
+//     console.log(true);
+//   }else{
+//     console.log(false)
+//   }
+// };
+
+function isUnderFive(number: number): boolean {
+  return number < 5;
+}
+
+// 2.
+function isEven(number: number): boolean {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// 3.
+function startsWithJ(string: string): boolean {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// 4.
+function isOldEnoughToDrink(person: { name: string; age: number }): boolean {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// 5.
+function isOldEnoughToDrive(person: { name: string; age: number }): boolean {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// 6.
+function isOldEnoughToDrinkAndDrive(person: {
+  name: string;
+  age: number;
+}): boolean {
```

Comment:
Since all 3 of these `function`s are expecting the same object, this would be a great example of when creating an `interface` would probably be a better option rather than typing the object directly